extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -145,32 +142,26 @@ export class VirtualListCell<ItemInfo extends VirtualListCellInfo> extends RX.Co
         const topValue = this._isVisible ? this._top : VirtualListCell._hiddenTopValue;
         this._topValue = RX.Animated.createValue(topValue);
 
-        const leftValue = props.left || 0;
-        this._leftValue = RX.Animated.createValue(leftValue);
-
         if (!props.isScreenReaderModeEnabled && !_isNativeMacOS) {
             // On native platforms, we'll stick with translate[X|Y] because it has a performance advantage.
             this._animatedStylePosition = RX.Styles.createAnimatedViewStyle({
                 transform: [{
                     translateY: this._topValue,
-                }],
-                left: this._leftValue,
+                }]
             });
         } else {
             // We need to work around an IE-specific bug. It doesn't properly handle
             // translateY in this case. In particular, if separate translations are used
             // within the item itself, it doesn't handle that combination.
             this._animatedStylePosition = RX.Styles.createAnimatedViewStyle({
-                top: this._topValue,
-                left: this._leftValue,
+                top: this._topValue
             });
         }
 
-        this._widthValue = RX.Animated.createValue(props.width || 0);
-
-        this._animatedStyleWidth = RX.Styles.createAnimatedViewStyle({
-            width: this._widthValue,
-        });
+        this._staticStylePosition = RX.Styles.createViewStyle({
+            width: this.props.width,
```

Comment:
This change will make the width (and left position) of items static after it's first loaded. VLV cells currently support changing widths via a prop. I'm not sure this is a desirable change, at the very least it's a breaking change. Typically width doesn't change on an item in common cases, but I can't speak for all the uses of VLV.